recipe/build.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# skip the creation of man pages by faking existance of help2man
+if [ `uname` == Darwin ]; then
+    export HELP2MAN=/usr/bin/true
+fi
+if [ `uname` == Linux ]; then
+    export HELP2MAN=/bin/true
```

Comment:
So I guess we gave up on `help2man`? That's fine with me. Will open an issue to track this though.